### DISCUSSION TITLE
Aging parameters

### DISCUSSION THREAD

**Discussion by agubucciarelli** [DLR-VSDC/IEEE-MVC-2023#15] [Q&A] [answered]
Hi! I noticed that in the FMU model there are 10 parametes for the aging model but in eq. (8) and (10) from the paper it can be seen that not all are used. Specifically theta_6, theta_9 and theta_10. Just out of curiosity, what are those parametes used for?

**Comment by rpintodecastro** [comment, marked as answer]
Hi,

For additional details on the aging model, please look into this paper:
https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9520124

These parameters are used to calculate N_knee according to eq. (17) - it is the fade effect point between short and long time aging effects

**Comment by JBRDLR** [comment]
We will update the chapter with beta 3 release:
![image](https://user-images.githubusercontent.com/110536277/211762972-5ce1656a-461d-4199-857b-7b0339846b02.png)

**Reply by woojmn** [reply]
In the aging equation, how are the average current(iavg) and average temperature(Tavg) defined or calaculated?
